packages/docs/guide/cli.md

```diff
@@ -4,13 +4,14 @@ title: Features | CLI
 
 # CLI
 
+In a project where Figus is installed, you can use the `figus` binary in your npm scripts, or run it directly with `npx @figus/cli`.
+
```

Comment:
Just my humble opinion, but this makes more sense to live here, right at the top. I personally also think all the documentation should reference the local binary, but that's more of my personal opinion 🙂